rest/route/distro.go

```diff
@@ -134,11 +174,6 @@ func (h *distroIDGetHandler) Factory() gimlet.RouteHandler {
 // Parse() fetches the distroId from the http request.
 func (h *distroIDGetHandler) Parse(ctx context.Context, r *http.Request) error {
 	h.distroId = gimlet.GetVars(r)["distro_id"]
-
-	if h.distroId == "" {
```

Comment:
Why remove this validation?

Reply:
I believe that the muxer cannot route here without populating the value of distro_id